src/auditor/cli.py

```diff
@@ -38,24 +43,66 @@ def cli():
         print(__doc__)
     else:
 
-        # report_dir = args['--save_report']
+        #: Logger that will gather the summary information.
+        summary_logger = logging.getLogger(__name__)
+        summary_logger.setLevel(logging.DEBUG)
 
-        cli_logger = logging.getLogger('auditor')
-        cli_logger.setLevel(logging.DEBUG)
-        detailed_formatter = logging.Formatter(
-            fmt='%(levelname)-7s %(asctime)s %(module)10s:%(lineno)5s %(message)s', datefmt='%m-%d %H:%M:%S'
-        )
-        cli_handler = logging.StreamHandler(stream=sys.stdout)
-        cli_handler.setLevel(logging.DEBUG)
-        cli_handler.setFormatter(detailed_formatter)
-        cli_logger.addHandler(cli_handler)
+        if args['spot']:
 
-        org_auditor = Auditor(cli_logger, args['--verbose'], args['ITEM'])
+            #: Only dump summary info to the console if verbose
+            #: Note: currently, org-wide checks (duplicate names) only reported if verbose
+            if args['--verbose']:
+                cli_handler = logging.StreamHandler(stream=sys.stdout)
+                cli_handler.setLevel(logging.DEBUG)
+                cli_formatter = logging.Formatter(
+                    fmt='%(levelname)-7s %(asctime)s %(module)10s:%(lineno)5s %(message)s', datefmt='%m-%d %H:%M:%S'
+                )
+                cli_handler.setFormatter(cli_formatter)
+                summary_logger.addHandler(cli_handler)
 
-        if args['--dry']:
-            org_auditor.check_items(args['--save_report'])
-        else:
-            org_auditor.check_items(report=False)  #: only do the fix report on a full run.
-            org_auditor.fix_items(args['--save_report'])
+            #: Set up org, check & fix items
+            org_auditor = Auditor(summary_logger, args['--verbose'], args['ITEM'])
+            if args['--dry']:
+                org_auditor.check_items(args['--save_report'])
+            else:
+                org_auditor.check_items(report=False)  #: only do the fix report on a full run.
+                org_auditor.fix_items(args['--save_report'])
```

Comment:
**suggestion**: this branching could be simplified if you gave `--save_report` a default value of `false` and then removed a duplicate `check_items` call; It would always run with the arg value. Then you only need to branch to run the `fix_items` method.

Reply:
I think the root problem here is high coupling due to poor design- the save report functionality is part of both `check_items` and `fix_items`, but would output to the same file for each call. Thus, if you're doing `fix_items` you still have to do `check_items` (to determine what needs to be fixed) but you don't want it to save the `check_items` report (which would contain the same data as will be in the fix report and would also trigger a rotate, thus cutting the expected dates covered by the rotating files in half). 

I should really be passing the check/fix info around as an object and then writing it to a file at a single point in the program. I hate to sound like a broken record, but I think this is another target for refactoring.